### PR TITLE
Styleguide buttons

### PR DESCRIPTION
**Pull request by tonydaly**
One of the issues that Tachyons introduces is that there's a lot of duplicate markup throughout our application e.g. for link and button styles. We want these to be consistent throughout the application so we're going to try building out all the components we need in our style guide, then using helpers to render these components everywhere in our application.

### CODE REVIEW

**Comment by tonydaly** [comment]
WIP screenshot

**Comment by tonydaly** [comment]
This should fix #370

**Comment by katbow** [comment]
Couple of comments:
The flash messages no longer look how they are supposed to. It's a [closed issue now](https://github.com/healthlocker/healthlocker/issues/364), so I don't want to change the styling of it after it was signed off on.
image

Another is the positioning in coping strategies has gone all weird. Back to toolkit also moves around depending on how many coping strategies there are - sometimes it's beside them and sometimes it's below them. 
image

Whereas previously back to toolkit was always at the bottom, and each strategy took up the whole width of the screen.

There's the same problem in /goals.

**Comment by tonydaly** [comment]
Good catch @katbow, strange as I haven't touched the styling for the flash messages. Will see what's going on.

File: web/views/component_helpers/button.ex [hunk 1]
```diff
@@ -0,0 +1,24 @@
+defmodule Healthlocker.ComponentHelpers.Button do
```
Comment (shouston3) [inline web/views/component_helpers/button.ex]: Ooh nice, @tonydaly, I didn't understand how hound would work before, but this is sweet

**Pull request merged**